mandatory/3-playing-computer.js

```diff
@@ -6,13 +6,13 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
-  2. Remove the line that throws the error.
+  1. This program throws an error. Why? (If you can't find it, try executing it). b is not defined.
+  2. Remove the line that throws the error. //console.log(b);
   3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  4. How many times is "f1" called? 2 times
+  5. How many times is "f2" called? 3 times
+  6. What value does the "a" parameter take in the first "f1" call? 8
```

Comment:
Good work Mireille. I have slightly difference answers for the questions 6 & 7 . What is the value of a.
It would be interesting to see the solutions to know what exactly the question was asking.
I had answers of 1 & 7, for a, for questions 6 & 7. But I was not completely confident in what the question was asking.

Reply:
Well Done Mireille . I was not sure about the meaning of outer variable (question 7) , so I put the value of a when called for the first time !

Reply:
Good work